spec/cc/engine/analyzers/node_translator_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "spec_helper"
+require "cc/engine/analyzers/node_translator"
+
+RSpec.describe(CC::Engine::Analyzers::NodeTranslator) do
```

Comment:
> My thinking was simple: the class translates a node

True, but not specific (IMO), not a huge deal. Maybe `NodeTranslator#to_sexp` would be a good compromise?